### DISCUSSION TITLE
feature:  Enhanced Main Menu / Login Layout

### DISCUSSION THREAD

**Discussion by TimtyG** [AscensionGameDev/Intersect-Engine#2405] [Ideas]
### Description

I propose an enhancement for the Main Menu, which is designed to create a more user-friendly and modern login experience. Currently, the Main Menu is giving us a traditional approach where users click a separate login button to access the login form. The feature request entails the following changes:

Attaching my wireframe that I'm trying to follow.
![New Wireframe 1](https://github.com/AscensionGameDev/Intersect-Engine/assets/36088127/c8632615-863e-4e66-b573-c4da7ca905e2)

### Intended Use-Case

Imagine a new player launching the game for the first time. With the current Main Menu, they are greeted with a somewhat old school design that requires them to click a login button to access the login form. This extra step can be confusing for newcomers, potentially leading to a less favorable first impression.

The login interface with username and password fields would be displayed at the Main Menu screen when the game is opened.
This eliminates the need for users to click a separate login button, making the login process more intuitive and efficient.

**How This Change Helps:**

_**Simplified Login:**_ New players can immediately log in without any additional clicks, providing a smoother onboarding experience.
_**Intuitive Navigation:**_ The new layout improves the navigation for both new and returning players by making essential functions readily accessible.

### Duplicate Check

- [X] This feature request is not a duplicate to the best of my knowledge.

**Comment by Arufonsu** [comment]
along with @WeylonSantana we agreed to do something like this:

![image](https://github.com/AscensionGameDev/Intersect-Engine/assets/17498701/8d4cc1c3-35ab-4853-b713-a93db57ac8c4)

**Comment by TimtyG** [comment]
Hey there, adding more samples that include all necessary buttons.

1. 
![image](https://github.com/user-attachments/assets/80ea2c55-5079-4d15-82cf-c9db9ad0e082)

2.
![image](https://github.com/user-attachments/assets/e8fb71a1-114b-4bc9-ab55-d76f92f68129)
3.
![image](https://github.com/user-attachments/assets/121c084b-818e-451e-9c77-6d136d4a836e)

**Comment by pandinocoder** [comment]
![image](https://github.com/user-attachments/assets/6b44b12a-9773-4f76-8101-33cfda2a10fa)
![image](https://github.com/user-attachments/assets/b75c0325-8707-49d5-b4a9-4d3a23116114)
![image](https://github.com/user-attachments/assets/a4c5aa43-8db9-4865-8911-8d6c9a74a88b)

"Forgot Password?" being a "text button" instead of a normal button and on the same line with Save Password would be better -- Save Password should also just be "Remember Me"

Login should be on the left and have a "pop" color while register is on the right with a more muted color

**Comment by TimtyG** [comment]
Probably not these colors, but like this?

![image](https://github.com/user-attachments/assets/9b44618f-ca88-47b2-ac93-bde3ab417005)
